[skip ci] Update INSTALLING.md, Galaxy 4U system reqs added

bbeggsTT opened the pull request:
### Ticket
[PR#27200](https://github.com/tenstorrent/tt-metal/pull/27200)

### Problem description
Galaxy 4U systems have specific reqs for firmware and tt-smi versions.

### What's changed
Warning has been added for specific Galaxy 4U system reqs.

### Checklist
- [ ] [All post commit](https://github.com/tenstorrent/tt-metal/actions/workflows/all-post-commit-workflows.yaml) CI passes
- [ ] [Blackhole Post commit](https://github.com/tenstorrent/tt-metal/actions/workflows/blackhole-post-commit.yaml) CI with demo tests passes (if applicable)
- [ ] [Model regression](https://github.com/tenstorrent/tt-metal/actions/workflows/perf-models.yaml) CI passes (if applicable)
- [ ] [Device performance regression](https://github.com/tenstorrent/tt-metal/actions/workflows/perf-device-models.yaml) CI passes (if applicable)
- [ ] (For models and ops writers) [Single-card demo tests](https://github.com/tenstorrent/tt-metal/actions/workflows/single-card-demo-tests.yaml) CI passes (if applicable) See [recommended dev flow](https://github.com/tenstorrent/tt-metal/blob/main/models/docs/MODEL_ADD.md#a-recommended-dev-flow-on-github-for-adding-new-models).
- [ ] [Galaxy quick](https://github.com/tenstorrent/tt-metal/actions/workflows/tg-quick-trigger.yaml) CI passes (if applicable)
- [ ] [TG demo tests, for Llama](https://github.com/tenstorrent/tt-metal/actions/workflows/tg-demo-tests.yaml) CI passes, if applicable, because of current Llama work
- [ ] (For runtime and ops writers) [T3000 unit tests](https://github.com/tenstorrent/tt-metal/actions/workflows/t3000-unit-tests.yaml) CI passes (if applicable, since this is run on push to main)
- [ ] (For models and ops writers) [T3000 demo tests](https://github.com/tenstorrent/tt-metal/actions/workflows/t3000-demo-tests.yaml) CI passes (if applicable, since this is required for release)
- [ ] New/Existing tests provide coverage for changes

bbeggsTT committed:
[skip ci] Update INSTALLING.md, Galaxy 4U system reqs added

Galaxy 4U systems have specific reqs for firmware and tt-smi versions. Warning has been added.

gsarabandoTT committed:
Add tt-installer flags

bbeggsTT committed:
Update INSTALLING.md

Blackhole entry added.

bbeggsTT committed:
Merge branch 'main' into bbeggsTT-patch-8

tt-rkim commented on INSTALLING.md:
```diff
@@ -23,6 +23,24 @@ curl -fsSL https://github.com/tenstorrent/tt-installer/releases/latest/download/
+> | Device               | OS              | Python   | Driver (TT-KMD)    | Firmware (TT-Flash)                        | TT-SMI                | TT-Topology                    |
+> |----------------------|-----------------|----------|--------------------|--------------------------------------------|-----------------------|--------------------------------|
+> | Galaxy (Wormhole 4U) | Ubuntu 22.04    | 3.10     | v1.33 or above     | fw_pack-80.10.1.0                          | v2.2.3 or lower       | v1.1.3, `mesh` config          |
+> | Blackhole            | Ubuntu 22.04    | 3.10     | v2.1.0 or above    | fw_pack-18.5.0.fwbundle (v18.5.0)          | v3.0.20 or above      | N/A                            |
```
```suggestion
> | Blackhole            | Ubuntu 22.04    | 3.10     | v2.1.0 or above    | fw_pack-18.7.0.fwbundle (v18.7.0)          | v3.0.20 or above      | N/A                            |
```

tt-rkim commented on INSTALLING.md:
```diff
@@ -23,6 +23,24 @@ curl -fsSL https://github.com/tenstorrent/tt-installer/releases/latest/download/
+> | Galaxy (Wormhole 4U) | Ubuntu 22.04    | 3.10     | v1.33 or above     | fw_pack-80.10.1.0                          | v2.2.3 or lower       | v1.1.3, `mesh` config          |
+> | Blackhole            | Ubuntu 22.04    | 3.10     | v2.1.0 or above    | fw_pack-18.5.0.fwbundle (v18.5.0)          | v3.0.20 or above      | N/A                            |
+
+- If required, add the following flags for specifying dependencies versions:
```
Should it be clearer that these are example versions?

bbeggsTT committed:
Update INSTALLING.md

Blackhole firmware version changes to 18.7.0
"versions" changed to "example versions"

bbeggsTT committed:
Update INSTALLING.md
